app/helpers/email_signup_helper.rb

```diff
@@ -0,0 +1,24 @@
+module EmailSignupHelper
+
+  def different_email_signup_path(atom_url)
```

Comment:
I think just going for `email_signup_path` would be better – "different" is a slightly confusing word. Different to what?